### PR TITLE
install instruction change

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: doc/faq.md [hunk 1]
```diff
@@ -9,25 +9,8 @@ Configuration can be put in either *xxx.cli.md* or *xxx.az.md* in *Azure/azure-r
...
-pip install azdev
-azdev setup -c -r ../azure-cli-extensions/
-
-# after the azdev setup you get the az
-
-# When the Azure CLI with new azure core feature released. you can skip all the step above. just run 
-pip install azure-cli 
-
-# Then run below command.
 az extension add --source=<cli-extension-whl-link-from-swagger-pipeline>
```
Comment (RodgeFu) [inline doc/faq.md:14]: please also include details about how to package

Comment (qiaozha) [reply doc/faq.md:14]: why? The packaging step is already done by pipeline automation

Comment (RodgeFu) [reply doc/faq.md:14]: oh, just realize this is for pipeline, could you also add one section for using generated code directly?

Comment (qiaozha) [reply doc/faq.md:14]: okay, add the instruction please have a look

File: doc/faq.md [hunk 2]
```diff
@@ -9,27 +9,36 @@ Configuration can be put in either *xxx.cli.md* or *xxx.az.md* in *Azure/azure-r
...
+``` yaml
+# how to build the generated file into a wheel file
 
-# after the azdev setup you get the az
+# 1. use python original way
+python setup.py sdist bdist_wheel
 
-# When the Azure CLI with new azure core feature released. you can skip all the step above. just run 
-pip install azure-cli 
+# 3. use azdev to build the wheel file
```
Comment (RodgeFu) [inline doc/faq.md:32]: 2? and user only needs to pick 1 or 3, right? then it's better to put them in one bullet, like
1. build whl file, there are two ways to do it
    a....
    b....

Comment (qiaozha) [reply doc/faq.md:32]: Yes but user can also choose not to build the wheel file

Comment (qiaozha) [reply doc/faq.md]: Changed. Please review again. thx

**Pull request merged**